numba/np/arraymath.py

```diff
@@ -632,77 +632,92 @@ def array_max_impl(arry):
     return impl_ret_borrowed(context, builder, sig.return_type, res)
 
 
-@lower_builtin(np.argmin, types.Array)
-@lower_builtin("array.argmin", types.Array)
-def array_argmin(context, builder, sig, args):
-    ty = sig.args[0].dtype
-
-    if (isinstance(ty, (types.NPDatetime, types.NPTimedelta))):
-        def array_argmin_impl(arry):
-            if arry.size == 0:
-                raise ValueError("attempt to get argmin of an empty sequence")
-            it = np.nditer(arry)
-            min_value = next(it).take(0)
-            min_idx = 0
-            if _is_nat(min_value):
-                return min_idx
+@register_jitable
```

Comment:
This function's code is unchanged, it's simply moved to the top-level whereas previously it was internal to `array_argmin`.

Reply:
Confirmed, thanks.